src/randoop/main/ClassTypeLoader.java

```diff
@@ -0,0 +1,134 @@
+package randoop.main;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+import randoop.types.TypeNames;
+import randoop.util.Files;
+
+import plume.EntryReader;
+
+/**
+ * ClassTypeLoader provides methods that allow a list of class names to be loaded
+ * as {@link Class<?>} objects.
+ * 
```

Comment:
Blank lines are ignored in the generated Javadoc.  You need to write `<p>` in order to start a new paragraph.  (I typically put it at the end of the previous paragraph rather than the start of the new one, but I don't write it at the end of the last paragraph.)